src/backend/TrafficCourts/Staff.Service/Authentication/Authentication.cs

```diff
@@ -20,8 +20,9 @@ public static void AddAuthentication(this IServiceCollection services, IConfigur
             {
                 configuration.Bind(_jwtBearerOptionsSection, options);
 
-                options.TokenValidationParameters.NameClaimType = "preferred_username";
-                options.TokenValidationParameters.RoleClaimType = _roleClaimType;
+                // removed until full authorization is enabled, otherwise this will case 403
```

Comment:
it is no longer necessary to comment out these as the 403 error will not be issued